doc/reference/reports.rst

```diff
@@ -176,9 +173,20 @@ For example, let's look at the Sale Order report from the Sale module::
     </template>
 
 
-The main template calls translate_doc with ``partner_id.lang`` as a parameter,
-which means it uses :ref:`a custom report model
-<reference/reports/custom_reports>` to access a ``res.partner`` record.
+The main template calls translate_doc with ``doc.partner_id.lang`` as a parameter, 
+which is the language of the partner_id of the document. This way, each Sale Order 
+will be printed in the language of the customer. If you wish to translate only the 
+body of the document, but keep the header and footer in a default language, you 
+could call the report's external layout this way::
+
+    <t t-call="report.external_layout">
+
+.. warning::
+
+    Please take note that this works only when calling external templates, you will not be
+    able to translate part of a document by setting a t-lang attribute on an xml node other 
+    than t-call.
```

Comment:
@dbo-odoo and thank you for discussing this topic with me.  Then let me suggest that in your warning you state that the current recommendation is that you "make part of your template external and translate it" as you stated above.  Then people know there is at least a solution not just "you will not be able to translate part of a document".

Maybe "participial translation of a document can be achieved by making that section part of an external template"